bitcoin-review-podcast/lightning-privacy-splice-panel.md

```diff
@@ -172,13 +184,13 @@ With Blended Path, you're finally getting good privacy with some trade-offs on p
 
 ## Blinded paths
```

Comment:
The sentence above also refers to blinded paths and was mistakenly transcribed to "blended path" You fixed it in the sentence right below but not in the one above.